src/data_preprocessing.py

```
@@ -0,0 +1,129 @@
+    print(f"Tanító mappák feldolgozása a(z) '{data_dir}' könyvtárból...")
+    print(f"Talált Sample mappák: {len(sample_folders)}")
+
+    for folder_name in tqdm(sample_folders, desc="Tanító mappák"):
+        folder_path = os.path.join(data_dir, folder_name)
+
+        # --- Címke kinyerése a mappanévből ---
+        try:
+            # Pl. 'Sample019' -> '019' -> 19
+            # Pl. 'Sample045' -> '045' -> 45
+            class_id_str = folder_name.replace('Sample', '').lstrip('0')
+            if not class_id_str: class_id_str = '0'  # Pl. 'Sample000' esete
```

Ruff (0.14.5)

Multiple statements on one line (colon)

(E701)